jenkins/scripts/bare_metal_lab/deploy-lab.yaml

```diff
@@ -7,6 +7,31 @@
     EPHEMERAL_CLUSTER: "minikube"
     EXTERNAL_VLAN_ID: 3
   tasks:
+    - name: set_fact
+      set_fact: 
+        node_ips: 
+          - "192.168.1.12"
+          - "192.168.1.15"
+        log_location: "/tmp/BMLlog"
+
+    - name: Delete existing ssh connection to HPE CLI
+      shell: 
+        cmd: |
+         killall ssh 
+         exit 0
+         EOT
+
+    - name: Start to capture log from virtual serial port 
+      shell: 
+        cmd: |
+          mkdir -p /tmp/BMLlog
+          nohup /bin/bash <<EOF 
+          ssh -o "KexAlgorithms diffie-hellman-group14-sha1" "{{ lookup('env', 'BML_ILO_USERNAME') }}"@{{ item }}  'vsp' > "{{ log_location }}/{{ item }}" &
```

Comment:
Is it possible to name the log file other than IP address? It can be difficult for the user to click and open to see the logs.